hyperopt/fmin.py

```diff
@@ -549,7 +549,7 @@ def fmin(
                 "There are no evaluation tasks, cannot return argmin of task losses."
             )
         if pickle_return is not None:
-            with open(pickle_return + '.pkl', 'wb') as out_file:
+            with open(str(pickle_return) + '.pkl', 'wb') as out:file:
```

Comment:
go back to previous and do double quotes

Reply:
Pull request closed/withdrawn.  Good to know about double quotes...I thought single quotes were the standard.